packages/ui/components/analytics/Analytics.module.css

```diff
@@ -0,0 +1,49 @@
+.container {
+	background: var(--White, #fff);
+	display: flex;
+	padding: 16px 120px;
+	flex-direction: column;
+	align-items: flex-start;
+	gap: 16px;
+	align-self: stretch;
+}
+.analyticSection {
+	opacity: 0.99;
+	display: flex;
+	flex-direction: column;

```

Comment:
Please group together common CSS properties in the parent element so that all children inherit the same.
This will reduce repititions.